src/api/config/options.yml.example

```diff
@@ -159,3 +159,32 @@ backup_user: tux
 backup_location: /home/tux
 backup_filename: obs_production.sql.bz2
 backup_port: 22
+
+# Rabbitmq based message bus
+#
+# ampq_namespace: 'opensuse.obs'
```

Comment:
do I overlook the default for this set in code?

Reply:
In case that someone removes it from the options.yml? That might be a good idea. Let me add it.